src/providers/completer/environment.ts

```diff
@@ -114,10 +240,60 @@ export class Environment {
             if (envList.includes(result[1])) {
                 continue
             }
+            const env: Suggestion = {
+                label: `${result[1]}`,
+                kind: vscode.CompletionItemKind.Module,
+                documentation: '`' + result[1] + '`',
+                filterText: result[1],
+                package: ''
+            }
 
-            envs.push(new vscode.CompletionItem(result[1], vscode.CompletionItemKind.Module))
+            envs.push(env)
             envList.push(result[1])
         }
         return envs
     }
+
+    entryEnvToCompletion(item: EnvItemEntry, type: EnvSnippetType): Suggestion {
+        const label = item.detail ? item.detail : item.name
+        const suggestion: Suggestion = {
+            label: item.name,
+            kind: vscode.CompletionItemKind.Function,
```

Comment:
Is it intentional to use `vscode.CompletionItemKind.Function` for environments here? Previously and other cases in the same file, `vscode.CompletionItemKind.Module` is used.

Reply:
This is a good question. I guess I took that part from 

https://github.com/James-Yu/LaTeX-Workshop/blob/cf0ec7c35f2aa1148f353e361ac5f7f58e09292a/src/providers/completer/command.ts#L455-L460

The value of `vscode.CompletionItemKind` seems a bit. The previous values were

- For true LaTeX commands: `vscode.CompletionItemKind.Function`
- For `\envname` snippet: `vscode.CompletionItemKind.Snippet`

https://github.com/James-Yu/LaTeX-Workshop/blob/a32dfdf/src/providers/completer/command.ts#L51-L58

- For environment completion: `vscode.CompletionItemKind.Module`

To stick to this, I propose to fix the value ``vscode.CompletionItemKind` based on the `EnvSnippetType`

- `AsName`: `vscode.CompletionItemKind.Module`. It just returns a list of environment names
- `AsCommand`: `vscode.CompletionItemKind.Snippet`. It returns commands of the form `\envname`.
- `ForBegin`: `vscode.CompletionItemKind.Module`. It is designed to complete `\begin{` with `envname}<some extra stuff>\n\end{envname}`.